<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceDataRIOM</class>
 <widget class="QWidget" name="DeviceDataRIOM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: black;
}
QPushButton
{
	font:18px;
	color:white;
	border-top: 2px solid white;	
	border-left: 2px solid white;
}
QLabel
{
color:rgb(255,255,255);
border:2px  solid white;
font:17px &quot;SImHei&quot;;
}
</string>
  </property>
  <widget class="QPushButton" name="btnRiom1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>RIOM1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRiom2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>RIOM2</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelName1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>67</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string>RIDO1</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelName2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>67</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string>RIDO1</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDriverSignal">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>450</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>219</x>
     <y>450</y>
     <width>111</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:yellow;
border:0px;</string>
   </property>
   <property name="text">
    <string>司控器信号</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNext">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>270</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>下一页</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPre">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>165</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>上一页</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPage">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>240</y>
     <width>67</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;
font:15px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
